packages/terra-list/tests/wdio/multi-select-list-spec.js

```diff
@@ -0,0 +1,207 @@
+      Terra.should.matchScreenshot('first and second item highlighted');
+
+      it('should highlight the third item upon enter', () => {
+        browser.keys(['Tab', 'Enter']);
+      });
+      Terra.should.matchScreenshot('first second and third item highlighted');
+
+      it('should unhighlight the second item upon enter', () => {
+        browser.keys(['Shift', 'Tab', 'Enter', 'Shift']);
+      });
+      Terra.should.matchScreenshot('first and third item highlighted');
+
+      it('should unhighlight the third item upon enter', () => {
+        browser.keys(['Tab', 'Enter']);
+      });
+      Terra.should.matchScreenshot('first item highlighted third item unhighlighted');
+    });
+
+    describe('Space Bar', () => {
+      before(() => browser.url('/#/raw/tests/terra-list/list/multi-select-list/default-multi-select-list'));
+
+      it('should highlight the selected items upon space keydown', () => {
+        browser.keys(['Tab', 'Space']);
+      });
+      Terra.should.matchScreenshot('first item highlighted');
+
+      it('should highlight the selected items upon space keydown', () => {
+        browser.keys(['Tab', 'Space']);
+      });
+      Terra.should.matchScreenshot('first and second item highlighted');
+
+      it('should highlight the selected items upon space keydown', () => {
+        browser.keys(['Tab', 'Space']);
+      });
+      Terra.should.matchScreenshot('first second and third item highlighted');
+
+      it('should highlight the selected items upon space keydown', () => {
+        browser.keys(['Shift', 'Tab', 'Space', 'Shift']);
+      });
+      Terra.should.matchScreenshot('first and third item highlighted');
+
+      it('should highlight the selected items upon space keydown', () => {
+        browser.keys(['Tab', 'Space']);
+      });
+      Terra.should.matchScreenshot('first item highlighted third item unhighlighted');
+    });
+    Terra.should.beAccessible();
+  });
+
+  describe('Multi Select List Max Count', () => {
+    before(() => browser.url('/#/raw/tests/terra-list/list/multi-select-list/max-count-multi-select-list'));
+    Terra.should.matchScreenshot();
+
+    it('should not highlight the third item', () => {
+      browser.keys(['Tab', 'Tab', 'Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('first and second item highlighted');
+
+    it('should unhighlight the second item', () => {
+      browser.click('ul li:nth-child(2)');
+    });
+    Terra.should.matchScreenshot('first item highlighted');
+
+    it('should highlight the third item', () => {
+      browser.keys(['Tab', 'Space']);
+    });
+    Terra.should.matchScreenshot('first and third item highlighted');
+
+    Terra.should.beAccessible();
+  });
+
+  describe('Multi Select List Focus', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-list/list/multi-select-list/default-multi-select-list');
+      browser.waitForExist('ul');
+      browser.keys(['Tab', 'Enter', 'Tab', 'Enter', 'Shift', 'Tab', 'Shift']);
```

Comment:
This is sending quite a few keys here, why `Shift`, `Tab`, `Shift`? it might be beneficial to add an inline comment here to  describe the outcome